strictdoc/export/html/_static/form.css

```diff
@@ -165,7 +165,17 @@ sdoc-form-row-main {
   display: flex;
   flex-direction: column;
   position: relative;
-  z-index: 2;
+  
```

Comment:
I just checked your branch, and I see that the tests pass, and it looks good visually even without this z-index.

Could you explain what will this do when we fix it? How will it improve the existing presentation?

Reply:
Actually, I see now. @mettta has to check.